docs/x509/reference.rst

```diff
@@ -499,6 +499,16 @@ X.509 Certificate Object
         :return bytes: The data that can be written to a file or sent
             over the network to be verified by clients.
 
+    .. method:: check_issued(issuer)
```

Comment:
Perhaps `check_issued_by` or `verify_issued_by`? Just "check issued" seems incomplete.

Reply:
I agree, but eventually this function calls `self._backend._lib.X509_check_issued`, and I would like to keep the naming convention as the library that we wrap. I see this in other places too.